talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -304,8 +304,6 @@ void P2PPeerConnectionChannel::SendSignalingMessage(
     const Json::Value& data,
     std::function<void()> on_success,
     std::function<void(std::unique_ptr<Exception>)> on_failure) {
-  if (ended_)
```

Comment:
Added this previously because I was thinking it caused segmentation faults as the signaling_sender_ was being deleted in the destructor of this class, but now that SendStop() must return before deleting the channel, this could be removed.